src/docs/manual/mockJOptionPane.java.txt

```diff
@@ -1,10 +1,8 @@
-package com.mypackage;
+package projectmock.javax.swing;
```

Comment:
Copy it from the default one.  Why do we need an example that is not the default one?
Using two (one of which is a .txt file that does not get tested) gives an opportunity for the two to fall out of sync.